anyway/widgets/widget_utils.py

```diff
@@ -319,3 +336,30 @@ def get_location_text(request_params: RequestParams) -> str:
         return f'{_("in segment")} {_(request_params.location_info["road_segment_name"])}'
     elif request_params.resolution == BE_CONST.ResolutionCategories.STREET:
         return f'{_("in street")} {request_params.location_info["street1_hebrew"]} {in_str}{request_params.location_info["yishuv_name"]}'
+
+
+__RESOLUTION_ACCURACY_VALUES: dict = {

```

Comment:
@ziv17 looks good! Perhaps we want these to be constants in BE_CONST?

Reply:
I think it depends on where does it come from, I mean is it a property of the accidents data? Will the other reports, like school reports, will also use it, or is it relevant only to Infographic Widgets?

Reply:
It is a property of the CBS accident data, and we might want to use it in other places than infographic Widgets